_layouts/home.html

```diff
@@ -17,5 +17,32 @@ <h2>Inspiring the next generation of engineers</h2>
     </div>
   </div>
 </div>
+<div class="panel panel-thin latest-content">
+  <div class="fixed-width">
+    <div class="content">
+      {% assign latest_post = site.posts.first %}
+      <h4>Latest Post:</h4>
```

Comment:
minor: thoughts on not including the colon here? These are headings rather than phrases, so I think it might look better without the colons.

Reply:
👍 2268b7c